dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -204,12 +225,22 @@ def profile(self):
 
         :return:
         """
+        num_most_common_words = self._num_most_common_words
+        if num_most_common_words is None:
+            num_most_common_words = len(self.word_count.keys())
```

Comment:
will be able to remove in future if `word_count` converts to `Counter`

Reply:
agreed